pkg/daemon/update.go

```diff
@@ -2230,6 +2230,8 @@ func (dn *CoreOSDaemon) applyLayeredOSChanges(mcDiff machineConfigDiff, oldConfi
 			dn.nodeWriter.Eventf(corev1.EventTypeNormal, "OSUpgradeSkipped", "OS upgrade skipped; new MachineConfig (%s) has same OS image (%s) as old MachineConfig (%s)", newConfig.Name, newConfig.Spec.OSImageURL, oldConfig.Name)
 		}
 	}
+	//if we're here, we've successfully pivoted, or pivoting wasn't necessary, so we reset the error gauge
```

Comment:
Apparently the verify job is complaining that you need a space between the `\\` and the `if` :sweat_smile:

Reply:
fixed (: